### ISSUE TITLE
Make Altered Vanilla Assets optional

### ISSUE THREAD

**Issue by Clashsoft** [MinestrapTeam/Minestrappolation-4#119]
Some changes to vanilla textures and block models introduced by Minestrappolation should be optional. In M3, this was accomplished by having a separate ZIP file that contained altered vanilla assets. This should probably continued and not forced upon the user.

**Comment by Delocuro** [comment]
Yep, that'd make sense. The only thing we'd want to check on is if there's some way to make it switch to the Minestrapp resource pack by default -- some of our custom textures and models make the vanilla stuff make more sense with their altered recipes/minestrapp tech uses, so having the pack on the first time the player uses the mod would help to at least show that it's there if they don't happen to check the resource pack menu.
